File: sync/src/main/scala/com/fan/mysql/event/impl/PreviousGtidsEvent.java
package com.dingcloud.dts.binlog.event.impl;

import com.dingcloud.dts.binlog.event.AbstractEvent;
import com.dingcloud.dts.binlog.event.EventHeader;

public class PreviousGtidsEvent extends AbstractEvent {

	private String gtidSet;

	public PreviousGtidsEvent(EventHeader header) {
		super(header);
	}

	public String getGtidSet() {
		return gtidSet;
	}

	public void setGtidSet(String gtidSet) {
		this.gtidSet = gtidSet;
	}

}

File: async/src/main/scala/com/fan/mysql/async/binlog/event/impl/PreviousGtidsEvent.scala
package com.fan.mysql.async.binlog.event.impl

import com.fan.mysql.async.binlog.event.{AbstractEvent, EventHeader}

class PreviousGtidsEvent(header: EventHeader) extends AbstractEvent(header) {
  private[this] var gtidSet: String = _

  def getGtidSet: String = gtidSet

  def setGtidSet(gtidSet: String): Unit = {
    this.gtidSet = gtidSet
  }
}
